md_rahaman/python/django/time_display/apps/time_display_app/views.py

```diff
@@ -0,0 +1,27 @@
+from django.shortcuts import render
+# import datetime
+import pytz
+# utc = pytz.utc
+# from datetime import datetime 
+import datetime
+from pytz import timezone
+fmt = "%B %d, %Y %I:%M %p"
```

Comment:
Would prefer for this variable to not be a global variable. Since all your of your view methods most likely won't use this variables. It would be better to create this variable only inside of the methods that require it.